model/task_queue_service_dependency.go

```diff
@@ -321,6 +321,37 @@ func (d *basicCachedDAGDispatcherImpl) FindNextTask(ctx context.Context, spec Ta
 				continue
 			}
 
+			if err = nextTaskFromDB.SetGenerateTasksEstimations(ctx); err != nil {
+				grip.Warning(message.WrapError(err, message.Fields{
+					"dispatcher": DAGDispatcher,
+					"function":   "FindNextTask",
+					"message":    "problem setting generate tasks estimations",
+					"task_id":    item.Id,
+					"distro_id":  d.distroID,
+				}))
+				continue
+			}
+
+			// Skip the task if it's estimated to create more tasks than the generate task limit.
+			generateTasksLimit := evergreen.GetEnvironment().Settings().TaskLimits.MaxPendingGeneratedTasks
```

Comment:
Can we use `GetConfig` here instead so it will respond without needing to restart the app servers if we change it?

Reply:
done